src/game/SpellEffects.cpp

```diff
@@ -4894,7 +4940,14 @@ void Spell::EffectDurabilityDamage(SpellEffectIndex eff_idx)
         return;
 
     if (Item* item = ((Player*)unitTarget)->GetItemByPos(INVENTORY_SLOT_BAG_0, slot))
+    {
         ((Player*)unitTarget)->DurabilityPointsLoss(item, damage);
+
+        ExecuteLogInfo info(unitTarget->GetObjectGuid());
+        info.durabilityDamage.itemEntry = item->GetProto()->ItemId;
+        info.durabilityDamage.unk = -1;
```

Comment:
Of course I've seen that. I meant it's not used in any way that can tell it's an item slot.